beetsplug/deezer.py

```diff
@@ -63,6 +63,25 @@ def func(lib, opts, args):
 
         return [deezer_update_cmd]
 
+    def fetch_data(self, url):
+        try:
+            response = requests.get(url, timeout=10)
+            response.raise_for_status()
+            data = response.json()
+        except (
+            requests.exceptions.ConnectionError,
+            requests.exceptions.RequestException,
+        ) as e:
+            self._log.debug("Error fetching data from {}\n Error: {}", url, e)
+            return None
+        else:
```

Comment:
And another revived comment:
https://github.com/beetbox/beets/pull/4984#discussion_r1402471354